docs/contents/dev-docs/testing.md

```diff
@@ -33,85 +26,25 @@ In summary, in order to test ACK you will need to have the following tools
 installed and configured:
 
 1. [Golang 1.14+](https://golang.org/doc/install)
+1. `make`
 1. [Docker](https://docs.docker.com/get-docker/)
 1. [kind](https://kind.sigs.k8s.io/docs/user/quick-start/)
-1. [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv1.html) version 1
 1. [jq](https://github.com/stedolan/jq/wiki/Installation)
-1. `make`
 
-To build and test an ACK controller with `kind`, execute the commands as 
-described in the following from the root directory of your
-[checked-out source repository](../setup/).
+To build and test an ACK controller with `kind`, execute the commands as
+described in the following from the root directory of the
+`github.com/aws/aws-controllers-k8s` repository.
```

Comment:
Should this link be updated to `github.com/aws-controllers-k8s/community` repository?